rebasehelper/tags.py

```diff
@@ -82,23 +84,49 @@ def _map_sections_to_parsed(cls, raw_content: SpecContent, parsed_content: SpecC
 
     @classmethod
     def _parse(cls, raw_content: SpecContent, parsed_content: SpecContent) -> Tuple[Tag]:
-        result = []
+        result: List[Tag] = []
         parsed_mapping = cls._map_sections_to_parsed(raw_content, parsed_content)
+        patch_index = 0
+        source_index = 0
```

Comment:
I still think `next_` prefix should be here.

Reply:
Also, why this order? Sources are usually before patches in a spec file, aren't they?